docs/source/parsing.md

```diff
@@ -49,6 +49,34 @@ The parsing process will write the following files:
   line N, with N starting from 0. Also, keep in mind that each ID corresponds with
   an ID of the `cw09b.documents` file.
 
+### Generating lexicon and document map
+Once the index has been parsed, you can generate the document map and lexicon which will be used at query time.
```

Comment:
Term and document lexicons are actually built at the end of `parse_collection`. So we might want to point that out and maybe emphasize that this command can be used to read or make lookups. I think it still make sense to explain the lexicon generation but maybe phrase it such that it's clear it can be done for any new-line delimited text file. Also maybe we should say that reverse lookup (string to ID) will work correctly only if the tokens are sorted.

Reply:
The reason for the last one is that we perform a binary search to find it.

Reply:
OK, have pushed some updates. Could you please take another look?